### PR TITLE
WIP: Add Links in EmotionResults to correct Learn page; fix antipattern in Play - closes #31 and #36

### LINKED ISSUE #31: Learn More About _emotion_ feature
Ref: comp

### LINKED ISSUE #36: Refactor: Play component
In app/components/Play/Play.js you're binding methods to pass down like so:
`<ImageCapture analyzeEmotions={this.analyzeEmotions.bind(this)}`
`getImageURL={this.getImageURL.bind(this)}`

This is considered an anti-pattern and has performance implications.  Here's an interesting read if you'd like to know more: 
https://medium.com/@esamatti/react-js-pure-render-performance-anti-pattern-fb88c101332f

A couple options are to bind the action in the constructor:
```
export default class Play extends Component {
  constructor() {
    super()
    this.state = {
      emotions: [],
      canvasURL: ''
    }
    this.analyzeEmotions = this.analyzeEmotions.bind(this)
  }
```

Or use an arrow function in the class property:
```
analyzeEmotions = (faceBlob) => {}
```

Binding in the constructor is supported in ES6 and recommended by React.  Using an arrow function in the class property is also recommended by React, but not supported in ES6, it's part of ES7 standard.  As such you'll need to modify your .babelrc to include `stage-2` presets. 

Some more reading/info:
 - https://medium.freecodecamp.com/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56
 - https://facebook.github.io/react/docs/handling-events.html

### PR DESCRIPTION
**Pull request by letakeane**
## What's this PR do?

_I added in ```<Link>``` tags to the EmotionResults component so users can learn more about the emotions they are displaying._

_I intentionally did not create a link for "neutral"._

_I also changed the antipattern of using ```function.bind(this)``` in the props to a dumb component._

## Learning

* _[This post about antipatterns for dumb components - thanks, @Jeff-Duke !](https://medium.com/@esamatti/react-js-pure-render-performance-anti-pattern-fb88c101332f)_
* _[This issue as a whole](https://github.com/letakeane/emotican-express/issues/36)_

## Where should the reviewer start?

_Take a look at the screenshot and let me know if the flow makes sense._

## Any background context you want to provide?
_I'm also trying to hook up a button for the intended emotion - the one selected on the left-hand side of the screen._

## Screenshots
Demo of the learn-more link:
Demo of link

### CODE REVIEW

**Comment by JohnBinning** [comment]
Looks pretty good Leta.  I might have missed something, but why are you filtering out contempt?  If you're not using it at all, it could potentially be easier to filter it out right at the response from the api call.

File: app/components/EmotionResults/EmotionResults.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+  const ifNotNeutralContemptRenderLink = (emotion) => {
+    if (emotion !== 'neutral' && emotion !== 'contempt') {
+      return (
+        <Link className='learn-more-link' to={`/learn/${emotion}`}>Learn more about {emotion}</Link>
+      )
+    }
+  }
+
   const showResults = () => {
     let emotionsForDisplay = analyzeResponse(results);
```
Comment (Tman22) [inline app/components/EmotionResults/EmotionResults.js]: I could be wrong but I think this should be a const. I don't think you're reassigning it.

Comment (letakeane) [inline app/components/EmotionResults/EmotionResults.js]: The results change, so the output of analyzeResponse changes ... but does that matter? I guess each time it's a new instance of emotionsForDisplay. I'm not mutating it once it's set.

**Comment by jbevis** [comment]
First of all the app looks incredible, it's a very clean UI and really cool feature. I think this is a great addition and adds another layer from the user can learn. Nice job. Is the info in the learn more feature also coming back from the api?

**Comment by letakeane** [comment]
@jbevis No, haha, I had to write it myself. I need to add references and further reading, still, but that's not MVP. I had a minor existential crisis when I was writing the happiness page - "What IS happiness?!"

**Pull request merged**